apps/ios/Shared/Views/Chat/ComposeMessage/ComposeView.swift

```diff
@@ -686,6 +712,31 @@ struct ComposeView: View {
         .frame(maxWidth: .infinity, alignment: .leading)
         .background(.thinMaterial)
     }
+    
+    
+    private func reportReasonView(_ reason: ReportReason) -> some View {
+        HStack {
```

Comment:
probably better to do the full string on each case, in some languages this may not even translate well